lib/filters/os_filter.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var OSFilter = function() {
```

Comment:
I think it would be really great if every one of these filters were accompanied by quick documentation of what the filter would look like in a test:
```
metadata: {
   requires: {
      os: 'something'
   }
}
```